Ines/index.js

```diff
@@ -0,0 +1,16 @@
+// 1
+let chrs = "ABCDEFGH";
+
+console.log("1 ==>", chrs.charAt(Math.random() * 8));
+
+//2
+let sy = "%&    § !* @= ";
+console.log("2 ==>", sy.charAt(Math.random() * 14));
```

Comment:
After some tries I get empty value, there was a typo in repository and Ghassan has fixed so let sy should be like this=>
let sy = " %&§!*@= ";
Means white spaces are just at the beginning and at the end, therefore you can use trim() instead of RegExp
let cleanSy = sy.trim();
let countSy = cleanSy.length;
let y = Math.floor(Math.random() * countSy );
console.log(cleanSy[y]);